lib/std/debug.zig

```diff
@@ -755,6 +755,8 @@ pub const OpenSelfDebugInfoError = error{
     UnsupportedOperatingSystem,
 };
 
+/// `allocator` should be an instance of `ArenaAllocator` to
+/// workaround https://github.com/ziglang/zig/issues/4353
```

Comment:
This comment is incorrect; it doesn't specifically have to be an instance of ArenaAllocator - it can be any kind of allocator that allows this callee to skip freeing.